src/types/__test__/1/collection.test.ts

```diff
@@ -50,6 +50,49 @@ const GET_VIEWER_COLLECTIONS = /* GraphQL */ `
     }
   }
 `
+
+const GET_COLLECTION_BY_ARTICLES = /* GraphQL */ `
+  query ($input1: NodeInput!, $input2: CollectionArticlesInput!) {
+    node(input: $input1) {
+      ... on Collection {
+        id
+        title
+        articles(input: $input2) {
+          pageInfo {
+            endCursor
+            hasNextPage
+          }
+          totalCount
+          edges {
+            node {
+              id
+            }
+          }
+        }
+      }
+    }
+  }
+`
+
+const GET_COLLECTION_PREV = /* GraphQL */ `
```

Comment:
looks like we can merge `GET_COLLECTION_BY_ARTICLES` and `GET_COLLECTION_PREV` into one query since they share the same input variables.

Reply:
good catch! updating